views/administration/forms/form-ldap-edit.hbs

```diff
@@ -6,73 +6,73 @@
 </div>
 
 <div class="form-group">
-    <label>Typ</label>
+    <label>{{$t "administration.global.label.type" }}</label>
     <select class="linked" name="ldaptype" data-linktext data-placeholder="LDAP-Typ auswählen">
             <option value="general">
-                Allgemein
+                {{$t "administration.ldapEdit.placeholder.general" }}
             </option>
     </select>
 </div>
 
 <div class="form-group">
-    <label>URL (ldaps ist verpflichtend)</label>
+    <label>{{$t "administration.ldapEdit.label.urlIdapIsMandatory" }}</label>
     <input class="form-control linked" name="ldapurl" type="text" placeholder="ldaps://" value="{{system.ldapConfig.url}}"/>
 </div>
 
 <div class="form-group">
-    <label>root-Pfad <i class="fa fa-question-circle" data-toggle="tooltip" title="Hauptpfad des LDAPs. Normalerweise DC=Domain,DC=Top-Level-Domain"></i></label>
+    <label>{{$t "administration.ldapEdit.label.rootPath" }} <i class="fa fa-question-circle" data-toggle="tooltip" title="Hauptpfad des LDAPs. Normalerweise DC=Domain,DC=Top-Level-Domain"></i></label>
     <input class="form-control linked" name="rootpath" type="text" value="{{system.ldapConfig.rootPath}}"/>
 </div>
 
 <div class="form-group">
-    <label>search-Nutzer <i class="fa fa-question-circle" data-toggle="tooltip" title="Voller Benutzername (inkl. root-Pfad) des Nutzers der Zugriff auf alle Nutzerinformationen hat"></i></label>
+    <label>{{$t "administration.ldapEdit.label.searchUsers" }} <i class="fa fa-question-circle" data-toggle="tooltip" title="Voller Benutzername (inkl. root-Pfad) des Nutzers der Zugriff auf alle Nutzerinformationen hat"></i></label>
     <input class="form-control linked" name="searchuser" type="text" value="{{system.ldapConfig.searchUser}}"/>
 </div>
 
 <div class="form-group">
-    <label>search-Nutzer Passwort</label>
+    <label>{{$t "administration.ldapEdit.label.seacrhUserpassword" }}</label>
     <input class="form-control linked" name="searchuserpassword" type="password" value="{{system.ldapConfig.searchUserPassword}}"/>
 </div>
 
 <div class="form-group">
-    <label>Nutzer-Pfad <i class="fa fa-question-circle" data-toggle="tooltip" title="Relativer Pfad vom root-Pfad zu dem LDAP-Knoten von dem alle Nutzer erreichbar sind. Mit der Trennung durch zwei Semikolons (;;) können auch mehrere relative Pfade zu LDAP-Nutzern angegeben werden."></i></label>
+    <label>{{$t "administration.ldapEdit.label.userPath" }} <i class="fa fa-question-circle" data-toggle="tooltip" title="Relativer Pfad vom root-Pfad zu dem LDAP-Knoten von dem alle Nutzer erreichbar sind. Mit der Trennung durch zwei Semikolons (;;) können auch mehrer relative Pfade zu LDAP-Nutzern angegeben werden."></i></label>
```

Comment:
title attribute i18n missing (please check the following occurrences in this file as well)

Reply:
done ;)